### ISSUE TITLE
Outdated Bundle version at README.md

### ISSUE THREAD

**Issue by inri13666** [eltrino/EltrinoOroCRMAmazonBundle#11]
When I use "README.md" instructions to install bundle with "orocrm-application", I got following error

```
> Oro\Bundle\MagentoBundle\Migrations\Schema\OroMagentoBundleInstaller
    ERROR: The column 'wsdl_url' on table 'oro_integration_transport' already exists.
```
This happens because I follow this [section](https://github.com/inri13666/EltrinoOroCRMAmazonBundle#composer)

**Comment by inri13666** [comment]
Added PR https://github.com/eltrino/EltrinoOroCRMAmazonBundle/pull/12

**Comment by odi-um** [comment]
Closed as issue it self have been fixed and pr #12 have been merged
